src/nvme/ioctl.h

```diff
@@ -4357,7 +4365,7 @@ struct nvme_zns_append_args {
 	int fd;
 	__u32 timeout;
 	__u32 nsid;
-	__u32 ilbrt;
+	__u64 ilbrt;
```

Comment:
Here too.

Reply:
Made changes to set the ilbrt field accordingly based on the version of the structure.